java/core/src/main/java/co/worklytics/psoxy/gateway/StorageEventRequest.java

```diff
@@ -0,0 +1,18 @@
+package co.worklytics.psoxy.gateway;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.io.InputStreamReader;
+
+@Builder
+@Getter
+public class StorageEventRequest {
```

Comment:
Wrapper for handling storage requests, here used only for AWS but in a future PR for GCP too.